packages/react-components/src/components/Dialpad/Dialpad.tsx

```diff
@@ -267,6 +267,15 @@ const DialpadContainer = (props: {
         onChange={(e: any) => {
           setText(e.target.value);
         }}
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        onClick={(e: any) => {
+          const input = e.target;
+          const end = input.value.length;
+
+          // Move focus to end of input field
+          input.setSelectionRange(end, end);
+          input.focus();
```

Comment:
This doesn't auto focus the dialpad text box right? Can a keyboard use tab in and out of the box as well?